src/main/resources/templates/email/programme-created/v1.0.0.html

```diff
@@ -0,0 +1,77 @@
+  <p><b>National Training Number (NTN)</b></p>
+  <p>Subject to you completing all actions on TIS Self Service, on the first day <span
+      th:text="${not #strings.isEmpty(startDate)} ? ${startDate} : _"></span> of your programme, you will receive your National Training Number (NTN). It is also vitally important you connect to your Postgraduate Dean for Revalidation and ensure you are connected to the correct Deanery as your Designated Body on GMC Online. Please do <b>not</b> do this before <span
+      th:text="${not #strings.isEmpty(startDate)} ? ${startDate} : _"></span>. It is essential you do not connect to your trust or employing body. The notice containing your NTN will explain how to do this.</p>
+  <p><b>Pre-employment checks and processes</b></p>
+  <p>Both NHS England and your employing organisation understand that you will be busy in this transition period to the start of your upcoming training programme, but we wish to emphasise the importance of engaging with all organisations with the onboarding process.</p>
+  <p>Your employing organisation will be contacting you about pre-employment checks from 12 weeks prior to starting. It is essential you avoid delaying these processes so as not to cause any disruption to your employment or training programme. <b>The Medical Staffing teams will be processing a large number of new starters to their organisation and so any delay may impact their ability to add you to their employment systems, such as payroll.</b></p>
+  <p>Please ensure you contact your employing organisation if you have not heard anything <b>8 weeks</b> prior to your start date. Details about who this will be, will be contained in the placement confirmation email that you will receive 12 weeks before starting.</p>
+  <p>This timescale is in line with the <a href="https://www.hee.nhs.uk/our-work/medical-recruitment/code-practice-medical-recruitment">code of practice</a> agreed in conjunction with the BMA and NHS Employers.</p>
+  <p>Their pre-employment checks will include:</p><ul>
+  <li>Your right to work</li>
+  <li>If you have a criminal record</li>
+  <li>Employment history and references</li>
+  <li>Professional Registration and qualifications</li>
+  <li>Fitness to work</li>
+</ul>
+  <p><b><u>ACTIONs</u></b></p>
+  <p><b>TIS Self-Service (TSS)</b></p>
+  <p>TIS Self-Service (TSS) gives you access to your training information. You will need to use TSS to submit essential forms and sign training agreements and complete actions that are presented on the Action Summary.</p>
+  <th:block th:switch="${isRegistered}"
+  ><p th:case="true">
+    We have a registered account for you on TIS Self Service against the email address <b><span th:text="${email}"></span></b>. To access details for onboarding onto your programme and actions to complete please log in to TIS Self Service: <a href="https://trainee.tis.nhs.uk/home">https://trainee.tis.nhs.uk/home</a>
```

Comment:
Get rid of `/home` in the URL, it's automatically appended but may not always be present if the app structure changes.

Reply:
I think this and a couple of other URLs also need some screen-readable text as well.